hathor/cli/run_node.py

```diff
@@ -109,12 +110,12 @@ def create_parser(cls) -> ArgumentParser:
         parser.add_argument('--config-yaml', type=str, help='Configuration yaml filepath')
         return parser
```

Comment:
Could you add a comment here pointing to `RunNodeArgs` so we don't forget to add new relevant arguments there?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/636/commits/1b1978be1ca0983ea1284e6298c1ea288f9964f9